java-imglib2/pom.xml

```
@@ -0,0 +1,160 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.scijava</groupId>
+		<artifactId>pom-scijava</artifactId>
+		<version>26.0.0</version>
```

```suggestion
		<version>31.1.0</version>
```

The latest pom-scijava is 31.1.0. https://github.com/scijava/pom-scijava/blob/master/pom.xml

---

Looks like I also need to get rid of `		<dependency>
			<groupId>net.imglib2</groupId>
			<artifactId>imglib2-ui</artifactId>
		</dependency>
```